powerline/segments/vim/plugin/ctrlp.py

```diff
@@ -0,0 +1,102 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+	import vim
+except ImportError:
+	vim = object()
+
+from powerline.bindings.vim import create_ruby_dpowerline
+
+
+def initialize():
+	global initialized
+	if initialized:
+		return
+	initialized = True
+	vim.command('echom 3210')
+	vim.command('let g:ctrlp_status = {}')
+	vim.command(
+		'''
+		execute 'function! Ctrlp_status_main(focus, byfname, regex, prev, item, next, marked)
+			let g:ctrlp_status = {\
+"focus": a:focus,\
+"byfname": a:byfname,\
+"regex": a:regex,\
+"prev": a:prev,\
+"item": a:item,\
+"next": a:next,\
+"marked": a:marked,\
+}
```

Comment:
And Vim is absolutely fine if you have tabs inside the dictionary literal. So there is no need in breaking indentation.